DSACMS/iv-cbv-payroll
# Get README Setup cleaned up, working

# No Ticket
This does not resolve any ticket. I just proactively made these changes while trying to get the repo to run locally.

## Changes
1. Updated the README in general, primarily with up-to-date instructions in the Setup section.
1. Added a new migration that enables the pgcrypto extension, which is required for the db:migrate step to work on a fresh install.

## Testing Instructions
1. Clone a fresh copy of this repo
2. Follow the instructions in the Setup section of the README
3. At the end, the site should be working!
